riscv-rt/README.md

```diff
@@ -0,0 +1,41 @@
+[![crates.io](https://img.shields.io/crates/d/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![crates.io](https://img.shields.io/crates/v/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![Build Status](https://travis-ci.org/rust-embedded/riscv-rt.svg?branch=master)](https://travis-ci.org/rust-embedded/riscv-rt)
```

Comment:
Is Travis CI actually used? Doesn't appear to be, I'm guessing this is just an artifact from the past.